chemprop/uncertainty/uncertainty_calibrator.py

```diff
@@ -138,8 +139,10 @@ def calibrate(self):
             self.calibration_predictor.get_uncal_preds()
         )  # shape(data, tasks)
         uncal_vars = np.array(self.calibration_predictor.get_uncal_vars())
-        targets = np.array(self.calibration_data.targets())
+        targets = np.array(self.calibration_data.targets(), dtype=float)
+        mask = np.array(self.calibration_data.mask())
         errors = uncal_preds - targets
+        errors = np.where(mask, errors, 0.)
```

Comment:
```suggestion
        targets = np.array(self.calibration_data.targets(), dtype=float)
        mask = np.array(self.calibration_data.mask())
        errors = uncal_preds - targets
        errors = np.where(mask, errors, 0.)
        uncal_vars = np.where(mask, uncal_vars, 0.)
```
Should we mask `uncal_vars` either? Because it will be passed to objective(), the `errors` in objective() has been masked, so I think  `uncal_vars` is supposed to be masked too.

Reply:
Oh, this is more of a problem than I thought. I thought I could ignore variance and the zero error would make the nll be zero. The variance shows up in both the numerator of one term and the denominator of another. So in order to make the nll zero, it would also cause a divided by zero error.

Same problem in tscaling really, though it's hidden inside another equation.

I'm going to need to loop both of these through the tasks to avoid this problem.